src/utils/ffmpeg.js

```diff
@@ -0,0 +1,156 @@
+import child_process from "node:child_process";
+import { EventEmitter } from "node:events";
+
+import { Logger } from "#src/utils/utils.js";
+import { recording, LOG_LEVEL } from "#src/config.js";
+
+const logger = new Logger("FFMPEG");
+
+/**
+ * hard-coded ffmpeg sdp fragments for layouts with 1...4 videos
+ * TODO make the right resizing and vstack/hstack params
```

Comment:
+TODO: 
if `1` `screen` (=limit), screen occupies most of the space and `n` `camera` fill space below (or sideways?)